eLearning-courseware/eLearning.jsonld

```diff
@@ -0,0 +1,801 @@
+        "definition": {
+          "en": "Records when the learner changes an answer."
+        },
+        "verb": "http://adlnet.gov/expapi/verbs/interacted",
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.correctResponsesPattern",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.choices",
+            "presence": "included"
+          },
+          {
+            "location": "$.result.response",
+            "presence": "included"
+          }
+        ]
+      },
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/questionAsked",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning/v1.0",
+        "prefLabel": {
+          "en": "Question asked"
+        },
+        "definition": {
+          "en": "The learner was asked a question."
+        },
+        "verb": "http://adlnet.gov/expapi/verbs/asked",
+        "rules": [
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.correctResponsesPattern",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.choices",
+            "presence": "included"
+          }
+        ]
+      },
+      {
+        "id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/templates/questionCommitted",
+        "type": "StatementTemplate",
+        "inScheme": "https://w3id.org/xapi/eLearning/v1.0",
+        "prefLabel": {
+          "en": "Question committed"
+        },
+        "definition": {
+          "en": "The learner committed an answer to a question."
+        },
+        "verb": "http://adlnet.gov/expapi/verbs/answered",
+        "rules": [
+          {
+            "location": "$.actor.mbox",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.id",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.description",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.correctResponsesPattern",
+            "presence": "included"
+          },
+          {
+            "location": "$.object.definition.choices",
+            "presence": "included"
+          },
+          {
+            "location": "$.result.success",
+            "any": [true, false],
```

Comment:
this is redundant, since if success is there it already must be true or false. Doesn't hurt anything though.